react/utils/page.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  replaceChannelSelectionInNavBar (loggedInChannel) {
```

Comment:
Shouldn't this design be obviated by addition of React/Redux? The component should be able to refresh based on external state change

Reply:
The reason for this was that html outside of the react publish component (the top nav bar) needed to be updated.  To solve this, I added a nav bar component that can draw from the store.